applications/tari_console_wallet/src/wallet_modes.rs

```diff
@@ -156,49 +158,65 @@ pub(crate) fn command_mode(
     wallet_or_exit(handle, cli, config, base_node_config, wallet)
 }
 
+pub(crate) fn parse_command_file(script: String) -> Result<Vec<CliCommands>, ExitError> {
+    let mut commands: Vec<CliCommands> = Vec::new();
+    let cli_parse_prefix = "tari_console_wallet --command n/a".to_string();
+
+    for command in script.lines() {
+        // skip empty lines and 'comments' starting with #
+        if !command.trim().is_empty() && !command.trim().starts_with('#') {
+            let command_trimmed = cli_parse_prefix.to_owned() + " " + command.trim();
```

Comment:
nit: a little more idiomatic
```suggestion
            let command_trimmed = format!("{} {}", cli_parse_prefix, command.trim());
```